client/base/help_options.cc

```diff
@@ -88,14 +88,14 @@ void Mysql::Tools::Base::Options::Help_options::print_usage()
 
   if (first_year_str == COPYRIGHT_NOTICE_CURRENT_YEAR)
   {
-    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE(COPYRIGHT_NOTICE_CURRENT_YEAR);
+    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE("2021");
   }
   else
   {
 #define FIRST_YEAR_CONSTANT "$first_year$"
     string first_year_constant_str= FIRST_YEAR_CONSTANT;
 
-    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE(FIRST_YEAR_CONSTANT);
+    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE("2021");
```

Comment:
Do not use magic number here.
Change all the 2021 to a macro definition.

Reply:
got it.  I have changed it